core-bundle/contao/classes/DataContainer.php

```diff
@@ -993,7 +993,16 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 				}
 				else
 				{
-					$return .= '<a href="' . $href . '" title="' . StringUtil::specialchars($config['title']) . '" onclick="Backend.getScrollOffset();return AjaxRequest.toggleField(this,' . ($icon == 'visible.svg' ? 'true' : 'false') . ')">' . Image::getHtml($state ? $icon : $_icon, $config['label'], 'data-icon="' . $icon . '" data-icon-disabled="' . $_icon . '" data-state="' . $state . '"') . '</a> ';
+					if (isset($config['titleDisabled']))
+					{
+						$titleDisabled = $config['titleDisabled'];
+					}
+					else
```

Comment:
I would prefer if the `DataContainerOperation` also handled `['label'][2]` as it already handles `['label'][0]` and `['label'][1]`. Then `$config['titleDisabled']` is always set and we do not need the if/else here.

Reply:
but the toggling is very specific to just one operation and no regular DCA operation ever has a `titleDisabled`?